service/src/test/java/uk/gov/hmcts/reform/fpl/model/PlacementTest.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.model.PlacementOrderAndNotices.PlacementOrderAndNoticesType.NOTICE_OF_HEARING;
+import static uk.gov.hmcts.reform.fpl.model.PlacementOrderAndNotices.PlacementOrderAndNoticesType.PLACEMENT_ORDER;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.unwrapElements;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+
+class PlacementTest {
+
+    @Test
+    void shouldReturnEmptyListWhenOnlyPlacementOrderIsPresentInListOfOrderAndNotices() {
+        Placement placement = Placement.builder()
+                .orderAndNotices(wrapElements(placementOrderAndNoticeOfType(PLACEMENT_ORDER)))
+                .build();
+
+        assertThat(unwrapElements(placement.removePlacementOrder().getOrderAndNotices())).isEmpty();
```

Comment:
this is not really correct test, it asserts that ordersAndNotices is right, but it does not check that rest of fields are preserved. Probably it should also verify that original object is untouched as this is quite critical here
Maybe:
 DocumentReference placementApplication = TestDataHelper.testDocument();

            Placement placement = Placement.builder()
                .application(placementApplication)
                .orderAndNotices(wrapElements(placementOrderAndNoticeOfType(PLACEMENT_ORDER)))
                .build();

            Placement expectedPlacement = Placement.builder()
                .application(placementApplication)
                .build();

            Placement actualPlacement = placement.removePlacementOrder();

            assertThat(actualPlacement).isEqualTo(expectedPlacement);

Reply:
this is a much better test 👍 , I shall follow for confidentialDoc removal test